data/json_converter.py

```diff
@@ -0,0 +1,40 @@
+"""
+    Converts the raw JSON data to dataframes 
+    and allows for the conversion to csv files. 
+"""
+
+from dataclasses import dataclass, field
+import pandas as pd
+from src.data.types import Airport
+
+
+@dataclass
+class DataConverter:
+    raw_data: dict[str, list[Airport]]
+    airports_df: pd.DataFrame = field(init = False)
+    runways: pd.DataFrame = field(init = False)
+    flights: pd.DataFrame = field(init = False)
+    air_quality: pd.DataFrame = field(init = False)
+
+    def __post_init__(self) -> None:
+        self.airports_df = pd.DataFrame(self.raw_data["airports"])
+        self.flights = self.airports_df[["flights"]]
+        self.runways = self.airports_df[["runways"]]
+        self.air_quality = self.airports_df[["air_quality"]]
+
+        self.airports_df = self.airports_df.drop(["flights","runways"],axis = 1)
+
+    def save_runways_to_csv(self) -> None:
+        self.runways.to_csv("/runways.csv", sep=",", index=False, encoding="utf-8")
```

Comment:
Would src/data/ be an appropriate place for these or would it be better to give the method a filename parameter?

Reply:
The issue isn't with the location it's with the path. On Linux based systems `/` is interpreted as the file system root which is ready only so this would probably fail. It's better to use a relative path e.g. `./runways.csv` so the file is saved in the cwd, or you could add a `file_name` parameter which defaults to `./runways.csv`